src/main/java/graphql/language/NodeChildrenContainer.java

```diff
@@ -99,6 +99,11 @@ public Builder replaceChild(String key, int index, Node newChild) {
             return this;
         }
 
+        public Builder removeChild(String key, int index) {
+            this.children.get(key).remove(index);
+            return this;
```

Comment:
This will blow up if the look up fails.  Is it likely to?  Should it be catered for?

Do we need a test for this?

Reply:
Should it assert if not found?

Reply:
I would not care at this point tbh: if the lookup fails you are doing something wrong higher up